<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 234, 234);</string>
  </property>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt;
border: 2px solid rgb(75, 75, 75);
border-top-width: 0px;
border-left-width: 0px;
border-right-width: 0px;
background-color: rgb(247, 247, 247);
border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToLogin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>377</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 12pt;
	color: rgb(253, 253, 253);
	background-color: rgb(122, 211, 116);
	border-radius: 8px;
}
QPushButton::hover {
	background-color: rgb(130, 224, 123);
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AutoLogin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 10pt;
	color: rgb(54, 54, 54);
}
QCheckBox::hover {
	color: rgb(70, 153, 255);
}</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RememberPassword">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	font: 10pt;
	color: rgb(54, 54, 54);
}
QCheckBox::hover {
	color: rgb(70, 153, 255);
}</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RetrievePassword">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt;
	background-color: rgb(0, 0, 0, 0);
	color: rgb(54, 54, 54);
}
QPushButton::hover {
	color: rgb(70, 153, 255);
}</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterAccount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 10pt;
	background-color: rgb(0, 0, 0, 0);
}
QPushButton::hover {
	color: rgb(70, 153, 255);
}</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Account">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	font: 12pt;
	border: 2px solid rgb(75, 75, 75);
	border-top-width: 0px;
	border-left-width: 0px;
	border-right-width: 0px;
	background-color: rgb(249, 249, 249);
	border-radius: 7px;
}
QComboBox::drop-down {
}
QComboBox::down-arrow {
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="HeadImg">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>157</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-radius: 10px;
	background: qlineargradient(x1:1, y1:1, x2:0, y2:0, stop:0 #41AEC6, stop:1 #E5E040);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FormatError">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt;
color: rgb(186, 60, 28);</string>
   </property>
   <property name="text">
    <string>请检查号码是否有误！
密码由8位及以上数字、字母或符号组成</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordError">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt;
color: rgb(186, 60, 28);</string>
   </property>
   <property name="text">
    <string>号码或密码错误，请再次尝试！</string>
   </property>
  </widget>
  <zorder>HeadImg</zorder>
  <zorder>Password</zorder>
  <zorder>ToLogin</zorder>
  <zorder>AutoLogin</zorder>
  <zorder>RememberPassword</zorder>
  <zorder>RetrievePassword</zorder>
  <zorder>RegisterAccount</zorder>
  <zorder>Account</zorder>
  <zorder>PasswordError</zorder>
  <zorder>FormatError</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
